src/entities/position.ts

```diff
@@ -305,6 +310,7 @@ export class Position {
    * @param amount1 token1 amount
    * @param useFullPrecision if false, liquidity will be maximized according to what the router can calculate,
    * not what core can theoretically support
+   * @returns the amount of liquidity for the position
```

Comment:
Here and a few other functions in this file don't have capitalization of "The". Should we standardize?